File: 760. Find Anagram Mappings/Solution.cpp
class Solution {
public:
    vector<int> anagramMappings(vector<int>& A, vector<int>& B) {
        map<int, int> hash_map;
        for (int i = 0; i < B.size(); i ++) {
            hash_map.insert(pair<int, int>(B[i], i));
        }
        vector<int> result;
        for (int i = 0; i < A.size(); i ++) {
            result.push_back(hash_map[A[i]]);
        }
        return result;
    }
};

File: 760. Find Anagram Mappings/Solution.py
class Solution(object):
    def anagramMappings(self, A, B):
        """
        :type A: List[int]
        :type B: List[int]
        :rtype: List[int]
        """
        hash_map = {val: index for index, val in enumerate(B)}
        return [hash_map[val] for val in A]
    